1-exercises/C-undefined-properties/exercise.js

```diff
@@ -35,4 +35,4 @@ let myPet = {
   },
 };
 
-console.log(myPet.getName());
+console.log(myPet.getName()); //has attempted to create a function element where the property should be//
```

Comment:
Please check what is missing in the function. 
This is quite interesting post about it: https://stackoverflow.com/questions/17337064/does-every-javascript-function-have-to-return-a-value